actions/lock-namespace/entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eu
+
+# use comma as input field separator
+PREVIFS=${IFS}
+IFS=','
+
+# try to lock a specific namespace
+while true; do
+    for NS in ${K8_NAMESPACES}; do
+        if kubectl get secrets -n "${NS}" "${NS}-inuse" > /dev/null 2>&1; then
+            echo "Namespace ${NS} in use -- Skipping"
+        else
+            echo "Namespace ${NS} not in use -- Locking"
+            kubectl create secret generic -n "${NS}" "${NS}-inuse" --from-literal=inuse="${NS}"
+            echo "NAMESPACEINUSE=${NS}" >> "$GITHUB_ENV"
```

Comment:
Is there a convention for documenting what env variables an action sets? Seems like it would be useful information to add to the action README.

Reply:
I added sections for inputs and outputs to the README.